src/finder.rs

```diff
@@ -141,6 +142,17 @@ impl LinkFinder {
         self
     }
 
+    /// Define whether URL schemes such as `https` are optional.
+    ///
+    /// By default only URLs having a scheme defined are found.
+    /// By setting this to `true` you make the scheme of URLs optional, to also find URLs like
+    /// `example.org`. For some URLs the used scheme is important, and making the scheme optional
+    /// may lead to finding a lot of false positive URLs.
+    pub fn url_optional_scheme(&mut self, url_optional_scheme: bool) -> &mut LinkFinder {
```

Comment:
Maybe we should name this similar to the existing option `email_domain_must_have_dot`, so `url_must_have_scheme`, which is `true` by default?

Reply:
Good point, should have thought about this before. Will change this soon.